2021-01-25 23:29:01,794: Unhandled exception in listener callback
Traceback (most recent call last):
  File "/home/wahyu/.virtualenvs/cv/lib/python3.6/site-packages/pynput/_util/__init__.py", line 211, in inner
    return f(self, *args, **kwargs)
  File "/home/wahyu/.virtualenvs/cv/lib/python3.6/site-packages/pynput/_util/xorg.py", line 471, in _handler
    self._handle(self._display_stop, event)
  File "/home/wahyu/.virtualenvs/cv/lib/python3.6/site-packages/pynput/keyboard/_xorg.py", line 564, in _handle
    self.on_press(key)
  File "/home/wahyu/.virtualenvs/cv/lib/python3.6/site-packages/pynput/_util/__init__.py", line 127, in inner
    if f(*args) is False:
  File "app.py", line 24, in on_press
    self.logging.info("Key pressed: {0}".format(key))
AttributeError: 'KeyboardListener' object has no attribute 'logging'
2021-01-25 23:29:52,462: Unhandled exception in listener callback
Traceback (most recent call last):
  File "/home/wahyu/.virtualenvs/cv/lib/python3.6/site-packages/pynput/_util/__init__.py", line 211, in inner
    return f(self, *args, **kwargs)
  File "/home/wahyu/.virtualenvs/cv/lib/python3.6/site-packages/pynput/_util/xorg.py", line 471, in _handler
    self._handle(self._display_stop, event)
  File "/home/wahyu/.virtualenvs/cv/lib/python3.6/site-packages/pynput/keyboard/_xorg.py", line 564, in _handle
    self.on_press(key)
  File "/home/wahyu/.virtualenvs/cv/lib/python3.6/site-packages/pynput/_util/__init__.py", line 127, in inner
    if f(*args) is False:
  File "app.py", line 24, in on_press
    self.logging.info("Key pressed: {0}".format(key))
AttributeError: 'NoneType' object has no attribute 'info'
